examples/celo-example/src/app/celo.auction.example.ts

```diff
@@ -100,9 +113,11 @@ export const celoAuctionExample = async () => {
       gasLimit: '40000',
```

Comment:
in general - dont provide fee object in examples - gasPrice could be small, gasLimit too big etc etc, it will be autocalculated

Reply:
yeah this was an old example, i just had to fix the type there

Reply:
so can we fix it even it is an old one?

Reply:
but celo is assigned to somebody else, it didn't make sense to me to spend time fixing this cause somebody else is working on it

Reply:
i would otherwise

Reply:
ok